pkgs/development/tools/benthos/default.nix

```
@@ -27,14 +28,14 @@ buildGoModule rec {
   ldflags = [
     "-s"
     "-w"
-    "-X github.com/benthosdev/benthos/v4/internal/cli.Version=${version}"
+    "-X github.com/redpanda-data/benthos/v4/internal/cli.Version=${version}"
   ];
 
   meta = with lib; {
     description = "Fancy stream processing made operationally mundane";
     mainProgram = "benthos";
     homepage = "https://www.benthos.dev";
```

```suggestion
    homepage = "https://docs.redpanda.com/redpanda-connect/about/";
```
Previous value was a redirect.